trade-finance-manager-api/src/v1/controllers/update-issued-facilities.js

```diff
@@ -30,54 +30,60 @@ const updatedIssuedFacilities = async (deal) => {
     const shouldUpdateFacility = (hasBeenIssued && !hasBeenAcknowledged);
 
     if (shouldUpdateFacility) {
+      let facilityPremiumSchedule;
+      let feeRecord;
+      let facilityUpdate;
+
       // update portal facility status
       const facilityStatusUpdate = CONSTANTS.FACILITIES.FACILITY_STATUS_PORTAL.ACKNOWLEDGED;
-      await api.updatePortalFacilityStatus(facilityId, facilityStatusUpdate);
-
-      // add acknowledged flag to Portal facility
-      const portalFacilityUpdate = {
-        hasBeenAcknowledged: true,
-      };
 
-      const updatedPortalFacility = await api.updatePortalFacility(facilityId, portalFacilityUpdate);
+      // Add `hasBeenAcknowledged` flag to non-GEF facility
```

Comment:
I'd say this instead just so it's totally clear

`
// add hasBeenAcknowledged flag and status to BSS/EWCS facilities
`